src/view/treeNodes/pullRequestNode.ts

```diff
@@ -138,11 +138,11 @@ export class PRNode extends TreeNode implements CommentHandler, vscode.Commentin
 			this._fileChanges = await this.resolveFileChanges();
 
 			if (!this._inMemPRContentProvider) {
-				this._inMemPRContentProvider = getInMemPRContentProvider().registerTextDocumentContentProvider(this.pullRequestModel.prNumber, this.provideDocumentContent.bind(this));
+				this._inMemPRContentProvider = getInMemPRContentProvider().registerTextDocumentContentProvider(this.pullRequestModel.number, this.provideDocumentContent.bind(this));
 			}
 
 			// The review manager will register a document comment's controller, so the node does not need to
-			if (!this.pullRequestModel.equals(this._prManager.activePullRequest)) {
+			if (!this.pullRequestModel.equals(this._prManager.activeItem)) {
```

Comment:
Since this file is also pull request specific, this should use `activePullRequest`